client/src/components/DownloadButton/DownloadButton.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Box, Button, Typography } from '@material-ui/core';
+
+const DownloadButton = (props) => {
+  return (
+    <div>
+      <Box display="flex" flexDirection="column" justifyContent="center" mx={2}>
+        <Button
+          color="secondary"
+          variant="contained"
+          disableElevation
+          style={{width: "400px"}}
+        >
+          <Typography variant="h6">
```

Comment:
```suggestion
          <Typography variant="h6" style={{ textTransform: 'none' }}>
```
This should fix the issue where the button text is automatically capitalizing.

Reply:
fixed this by editing theme (MuiButtonLabel)